modules/metrics/src/main/java/org/apache/ignite/internal/metrics/sources/JvmMetricSource.java

```diff
@@ -117,11 +127,36 @@ public String name() {
                         () -> nonHeapMemoryUsage.get().getMax()
                 ));
 
+        for (GarbageCollectorMXBean gcMxBean : gcMxBeans) {
+            metrics.put(
+                    "gc." + nameForMetricName(gcMxBean) + ".CollectionCount",
+                    new LongGauge(
+                            "gc." + nameForMetricName(gcMxBean) + ".CollectionCount",
+                            "Total number of collections that have occurred. -1 if the collection count is undefined for this collector.",
+                            gcMxBean::getCollectionCount
+                    )
+            );
+
+            metrics.put(
+                    "gc." + nameForMetricName(gcMxBean) + ".CollectionTime",
```

Comment:
Maybe: `"gc." + nameForMetricName(gcMxBean)` to method?

Reply:
This method got removed